### PR TITLE
Adds RFC for cabbage roadmap for start of 2022

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Closes https://github.com/giantswarm/giantswarm/issues/20883
Towards https://github.com/giantswarm/giantswarm/issues/18676

This RFC describes the roadmap for Team Cabbage for the first part of 2022.

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
Starting with reviews from cabbage, then i'll open it up to sig product.

**Comment by JosephSalisbury** [comment]
yolo, let's go sig product

File: cabbage-roadmap-2022-1/README.md
Comment (puja108) [inline cabbage-roadmap-2022-1/README.md:35]: I think we could mention here already that one of the main use cases is supporting better multi-tenancy and separation of concerns within the use case of exposing services.

File: cabbage-roadmap-2022-1/README.md
Comment (JosephSalisbury) [reply cabbage-roadmap-2022-1/README.md:3]: Yeah, I don't know if it's just for Q2, or Q2 or Q3, or if we'll re-evaluate everything next month. So, staying vague intentionally :D

File: cabbage-roadmap-2022-1/README.md
Comment (JosephSalisbury) [reply cabbage-roadmap-2022-1/README.md:19]: I didn't intend for any specific intention w/ 'microservices' - i'm thinking APIs in a very general sense

**Review by puja108 — COMMENTED** [review]
minor nitpicks, otherwise LGTM, nice to see old school roadmap dos making a comeback

**Review by ubergesundheit — APPROVED** [review]

File: cabbage-roadmap-2022-1/README.md
Comment (JosephSalisbury) [reply cabbage-roadmap-2022-1/README.md:19]: moved to 'API services', I think that's generic enough

File: cabbage-roadmap-2022-1/README.md
Comment (JosephSalisbury) [reply cabbage-roadmap-2022-1/README.md:35]: Added a bit lower

**Comment by JosephSalisbury** [comment]
@puja108 please take another look and hit me w/ an approve if you like

@weatherhog i'd also like comments / an approve from you too before merging

**Review by puja108 — APPROVED** [review]

**Review by weatherhog — APPROVED** [review]

File: cabbage-roadmap-2022-1/README.md
Comment (JosephSalisbury) [reply cabbage-roadmap-2022-1/README.md:103]: I think I'd leave that to be decided somehow - it feels like part of the design of the solution.

**Pull request merged**
